osbs/api.py

```diff
@@ -395,13 +393,21 @@ def _create_build_config_and_build(self, build_request):
             logger.debug('Changed parent ImageStreamTag? %s', changed_ist)
 
         if triggers:
-            existing_bc['spec']['triggers'] = triggers
-            self.os.update_build_config(build_config_name, json.dumps(existing_bc))
```

Comment:
clean triggers is cleaning not ImageChange triggers from existing config

this lines were suppose to update buildconfig with new triggers